README.md

```diff
@@ -32,16 +32,22 @@ Or, for development, build css and restart server on file changes:
 $ npm run start:dev
 ```
 
-To run locally you also need the database, mock server, and the auth server. These can all be started by running
+To run locally you need:
+- database
+- redis
+- auth server
+- nomis
+- community api
+- probation teams
+- gotenberg
+- nomis-user-roles-api
 
-```
-npm run start-mocks
-docker-compose up
-```
+You can run local database, redis and configure the application to point to the T3/dev environment for other services.
+Or these can be run locally using `docker-compose` and starting `mock-services`.
+Or you can point to mock-services for a subset of services and T3/dev for others..
```

Comment:
Putting a more emphasis on running against T3 rather than struggling with a load of different local services